packages/utilities/shared-icons/src/dustbin.tsx

```diff
@@ -0,0 +1,25 @@
+import {IconSvgProps} from "./types";
+
+export const DustbinIcon = (props: IconSvgProps) => (
```

Comment:
rename to `TrashIcon` since it's from `vuesax`. Also rename the file name.

Reply:
ok,i will change it with the case without label